src/release-notes/api.md

```diff
@@ -0,0 +1,30 @@
+---
+title: MyParcel API release notes
+sidebar: auto
+---
+Hieronder kan je de release notes vinden van de MyParcel API. Als er vragen over een release zijn, dan kan je contact opnemen via ons [slack kanaal](https://join.slack.com/t/myparcel-dev/shared_invite/enQtNDkyNTg3NzA1MjM4LWQ5MWE5MTQ3MDg4YjU5NzdjYjk0OTY1ZDJiYjU5YzJjNzk3Yzk3NGY0OWFkZDU4MDYwZDEyZDlhZTgzOWM1MjI/).
+
```

Comment:
moeten deze release notes niet in het Engels? onze api documentatie is ook in het engels, vind het wel inconsistent zo